### PR TITLE
Aria-labelledby through h2 in modals

### PR DESCRIPTION
**Pull request by pascalwengerter**

### CODE REVIEW

File: src/components/OcModal.vue [hunk 1]
```diff
@@ -11,7 +11,7 @@
       >
         <div class="oc-modal-title">
           <oc-icon v-if="icon" :name="icon" :variation="variation" />
-          <h2 v-text="title" />
+          <h2 id="oc-modal-label" v-text="title" />
```
Comment (kulmann) [inline src/components/OcModal.vue:14]: Why not `oc-modal-title` as id? 🤔

Comment (pascalwengerter) [reply src/components/OcModal.vue:14]: Also fine for me, saw the a11y example use `-label` and went with it..

Comment (kulmann) [reply src/components/OcModal.vue:14]: I'd prefer `oc-modal-title` 😁

Comment (pascalwengerter) [reply src/components/OcModal.vue]: `oc-modal-title` it is, then

**Review by kulmann — APPROVED** [review]
👍

**Pull request merged**